core.md

```diff
@@ -274,42 +367,59 @@ before, but with a different offset. The offset is now 16, because it
 points at the `to` point (and each point consists of two `float64`s,
 which are 8 bytes each).
 
-Unlike arrays and structs, accessing a field of primitive type does
+Unlike for structs, accessing a field of primitive type does
```

Comment:
this is actually somewhat grammatically ambiguous (as was the previous formulation). Perhaps: "accessing a field of primitive type does not return a typed object, in contrast to fields of struct type."